doc/source/user_guide/dsintro.rst

```diff
@@ -650,6 +650,13 @@ row-wise. For example:
 
    df - df.iloc[0]
 
+Use .add(array, axis=0) to apply row-wise broadcasting when the array length matches the number of rows — 
```

Comment:
the next line says that this is all already covered in ``:ref:\`flexible binary operations <basics.binop>\``